docs/references/kontena-yml.md

```diff
@@ -178,6 +178,16 @@ hooks:
       cmd: grunt dist
 ```
 
+**post_install**
+
+Display a message in the CLI after the service has been installed.
```

Comment:
Please have some explicit reference to `kontena stack install`, because that's the only CLI command that uses these.